examples/magnum/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-magnum)
+
+# The part below is adapted from
+# https://github.com/mosra/magnum-examples/blob/master/src/primitives/CMakeLists.txt
+
+# CMake policies. 0025 needs to be before project(), so putting all there.
+# Use AppleClang instead of Clang on Apple
+if(POLICY CMP0025)
+    cmake_policy(SET CMP0025 NEW)
+endif()
+# Don't treat imported targets with :: as files
+if(POLICY CMP0028)
+    cmake_policy(SET CMP0028 NEW)
+endif()
+# Enable MACOSX_RPATH by default
+if(POLICY CMP0042)
+    cmake_policy(SET CMP0042 NEW)
+endif()
+# Quoted variables should not be dereferenced
+if(POLICY CMP0054)
+    cmake_policy(SET CMP0054 NEW)
+endif()
```

Comment:
Magnum (and Corrade) now depends on CMake 3.1, which includes all the policies above, so none of the `cmake_policy()` calls are needed anymore (and `cmake_minimum_required()` should be set to 3.1 now -- the same for Corrade, I think). Current version, for reference: https://github.com/mosra/magnum-examples/blob/master/src/primitives/CMakeLists.txt